{
  "success": false,
  "error": "Traceback (most recent call last):\n  File \"/home/kabir/.local/lib/python3.10/site-packages/slither/__main__.py\", line 744, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/home/kabir/.local/lib/python3.10/site-packages/slither/__main__.py\", line 87, in process_all\n    ) = process_single(compilation, args, detector_classes, printer_classes)\n  File \"/home/kabir/.local/lib/python3.10/site-packages/slither/__main__.py\", line 70, in process_single\n    slither = Slither(target, ast_format=ast, **vars(args))\n  File \"/home/kabir/.local/lib/python3.10/site-packages/slither/slither.py\", line 95, in __init__\n    parser.parse_top_level_from_loaded_json(ast, path)\n  File \"/home/kabir/.local/lib/python3.10/site-packages/slither/solc_parsing/slither_compilation_unit_solc.py\", line 238, in parse_top_level_from_loaded_json\n    _handle_import_aliases(symbol_aliases, import_directive, scope)\n  File \"/home/kabir/.local/lib/python3.10/site-packages/slither/solc_parsing/slither_compilation_unit_solc.py\", line 50, in _handle_import_aliases\n    and \"name\" in symbol_alias[\"foreign\"]\nTypeError: argument of type 'int' is not iterable\n",
  "results": {}
}